content-model/articles.json

```diff
@@ -96,6 +96,32 @@
               }
             }
           },
+          "iframe" : {
+            "type" : "Slice",
+            "fieldset" : "Iframe",
+            "non-repeat" : {
+              "launchButtonText" : {
```

Comment:
Why is this text not consistent?

Reply:
My thinking was that the iframe could contain anything – this is a way to indicate what's going to be launched.

Reply:
I'd leave it as "Launch" or "Play" until we need it... your call.